docs/docs/adapters/edgedb.md

```diff
@@ -0,0 +1,269 @@
+---
+id: edgedb
+title: EdgeDB
+---
+
+# EdgeDB
+
+To use this Adapter, you need to install `edgedb`, `@edgedb/generate`, and the separate `@next-auth/edgedb-adapter` package:
+
+```bash npm2yarn2pnpm
+npm install next-auth edgedb @next-auth/edgedb-adapter
+npm install @edgedb/generate --save-dev
+```
+
+## Installation
```

Comment:
It might be mroe future-proof to link out to our quickstart guide. Something like 
"Follow the EdgeDB quickstart to install the EdgeDB CLI and initailize a project". That way you don't need to worry about keeping all this information up-to-date if/when workflows change.